codeforboston/maple
# feat: bill follow and unfollow functionality to testimony page

# Summary

#1652 A functional click event with a request and resposne has been added to the Follow bill button in the menu on the Testimony Detail page for each testimony.

# Checklist


# Screenshots

<img width="614" alt="Screenshot 2024-12-17 at 5 44 51 PM" src="https://github.com/user-attachments/assets/021b46b1-8fb5-41b3-bbb0-003f980b4f24" />

<img width="1585" alt="Screenshot 2024-12-17 at 5 44 46 PM" src="https://github.com/user-attachments/assets/8f8ddf7d-b16a-4d83-9b3d-180484f93388" />


# Known issues


# Steps to test/reproduce

1. Go to the list of testimonies page, accessed from either the nav option or the page main section link option
2. Click on and go to the Testimony Detail page
3. Click on Follow button, should now read "Unfollow", action should work for following/unfollowing. See testimonies linked to the same bill should read the same for the button.
